star_test.ipynb

```diff
@@ -57,7 +57,7 @@
     "from astropy import units as u, constants as const\n",
```

Comment:
<p>I think it will be nice to explain why we chose bb at 15. If you want, you can use this code I had written in my notebook:</p><pre class="ql-syntax" spellcheck="false">t_at_depth0 = marcs_model[marcs_model.depth==0].t.values[0]
t_at_depth0_idx = np.where(marcs_model.t.values==t_at_depth0)[0][0]
</pre><p> </p><p><code>t_at_depth0_idx</code> turns out to be 15.<span class="ql-cursor">﻿</span>
</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>